### PR TITLE
Issue #4303 Add Fairwood

### PR DESCRIPTION
**Pull request by TheDynamicPunk**
Added Fairwood

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -1185,6 +1185,20 @@
...
+    {
+      "displayName": "Fairwood",
+      "id": "fairwood-7e2e93",
+      "locationSet": {"include": ["cn", "hk"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Fairwood",
+        "brand:wikidata": "Q5430935",
+        "brand:wikipedia": "en:Fairwood (restaurant)",
+        "cuisine": "chinese and american",
```
Comment (bhousel) [inline data/brands/amenity/fast_food.json:1197]: The correct way to have multiple cuisines in OpenStreetMap is to separate them with a semicolon..
```suggestion
        "cuisine": "chinese;american",
```

**Review by bhousel — CHANGES_REQUESTED** [review]
Looks mostly good to me, just one minor change..

File: data/brands/amenity/fast_food.json (hunk 1 above)
Comment (TheDynamicPunk) [reply data/brands/amenity/fast_food.json]: Thanks for letting me know! I have updated this one and will take care of it in the future!

**Comment by TheDynamicPunk** [comment]
@bhousel I've updated it and committed the changes!

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks @TheDynamicPunk 👍

**Pull request merged**